packages/tds-migration/src/main/resources/application.properties

```diff
@@ -0,0 +1,15 @@
+########################################################################################################################
+# Database configuration
+########################################################################################################################
+spring.datasource.url=jdbc:postgresql://localhost:5430/tds
```

Comment:
Does `localhost` will work on k8 on AWS?

Reply:
running this on kubernetes you would overwrite just like we do in hmi-server